src/Order.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useState } from "react";
+
+const Order = ({ orderType }) => {
+  //   const orderType = props.orderType;
```

Comment:
Make sure you remove any rogue comments like this - don't think you need this one anymore